packages/web-components/src/components/va-select/va-select.scss

```diff
@@ -11,12 +11,16 @@
 @import '../../mixins/uswds-error-border.scss';
 @import '../../global/formation_overrides';
 
-:host {
-  color: var(--color-base);
+:host([uswds]) {
+  color: var(--v3-color-base-darkest);
   display: block;
   font-family: var(--font-source-sans);
 }
```

Comment:
I think you meant to make two separate `:host` styles. Otherwise, as it is now, all of those styles will only apply to v3.

So I believe you want to add one for v3:

```css
:host([uswds]) {
  color: var(--v3-color-base-darkest);
}
```

And one for v1 and v3 (this is what was there originally):

```css
:host {
  display: block;
  font-family: var(--font-source-sans);
}
```